addons/mail/static/src/discuss/call/common/call_settings.xml

```diff
@@ -36,13 +36,17 @@
                 <span t-if="store.settings.use_push_to_talk and !isMobileOS and !pttExtService.isEnabled" class="small text-muted fst-italic mb-3" t-out="pttExtService.downloadText"/>
             </div>
             <div class="d-flex flex-column">
-                <label t-if="!store.settings.use_push_to_talk" class="d-flex flex-column flex-wrap flex-grow-1 align-items-start" title="Voice detection threshold" aria-label="Voice detection threshold">
+                <t t-if="!store.settings.use_push_to_talk">
                     <span class="me-2 my-1 text-truncate text-wrap">Voice detection threshold</span>
-                    <div class="d-flex w-100 align-items-center">
-                        <input class="flex-grow-1 form-range" type="range" min="0.001" max="1" step="0.001" t-att-value="store.settings.voiceActivationThreshold" t-on-change="onChangeThreshold"/>
-                        <span class="p-1 text-end"><t t-out="Math.floor(store.settings.voiceActivationThreshold * 100)"/>%</span>
+                    <div class="d-flex align-items-center w-100 h-100">
+                        <button class="btn btn-primary" t-on-click="testMicrophoneVolume" t-att-disabled="!state.isTestMicrophoneReady">
+                            <t t-if="audioTrack">Stop</t><t t-else="">Test</t>
+                        </button>
```

Comment:
Note that due to label of button changing, the range width changes. This makes visual inconsistent between non-testing and testing different. Not a huge deal, but can be frustrating that the threshold position changes, as if there was some auto-adjusting fighting the user manual setting.

Reply:
done